plugins/pur/core/src/main/java/org/pentaho/di/repository/pur/JobDelegate.java

```diff
@@ -180,43 +180,18 @@ public JobDelegate( final Repository repo, final IUnifiedRepository pur ) {
   // ~ Methods =========================================================================================================
   @SuppressWarnings( "unchecked" )
   @Override
+  @Deprecated // Shared Object reads should now go through a SharedObjectsIO
```

Comment:
I think PUC uses these classes. Is this tested from PUC?

Reply:
All the calls to loadSharedObjects I can find are in kettle. I think you are probably right about their use for import/export. The only remaining call I can see is in RepositoryImporter. 

*saveSharedObjects* is still used in a couple more places, though, so we need to be more careful of that. I think we should leave this deleted for now, and revisit it in the story where we do import/export.

Reply:
I think it is also called during execution from PUC

Reply:
That should now also go through the bowl. We should probably have that as a story, though.